test/test_sparse_semi_structured.py

```diff
@@ -853,6 +853,7 @@ class TestSparseSemiStructuredCUTLASS(TestCase):
          - torch._sparse_semi_structured_linear
     """
     def setUp(self):
+        SparseSemiStructuredTensor._FORCE_CUTLASS = True
```

Comment:
Do you not need to restore this in the teardown?

Reply:
Good idea!